### PR TITLE
Json support schemaverse

### PR DESCRIPTION
**Pull request by shohamroditimemphis**

### CODE REVIEW

File: ui_src/src/domain/schema/components/createSchema/index.js [hunk 1]
```diff
@@ -310,47 +432,56 @@ function CreateSchema() {
...
-                                colorType="white"
-                                radiusType="circle"
-                                backgroundColorType="purple"
-                                fontSize="12px"
-                                fontFamily="InterSemiBold"
-                                isLoading={validateLoading}
-                                disabled={formFields?.schema_content === ''}
-                                onClick={() => handleValidateSchema()}
-                            />
+                            {formFields.type === 'Json' ? (
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/createSchema/index.js:435]: The if statement needs to be only on the Convert button

File: ui_src/src/domain/schema/components/createSchema/index.js [hunk 2]
```diff
@@ -310,47 +432,56 @@ function CreateSchema() {
...
+                                <>
+                                    <Button
+                                        width="170px"
+                                        height="34px"
+                                        placeholder="Convet json to json schema"
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/createSchema/index.js:440]: "Convert to JSON schema"

File: ui_src/src/domain/schema/components/schemaBox/style.scss [hunk 3]
```diff
@@ -6,6 +6,10 @@
     width: 330px;
     height: 183px;
     cursor: pointer;
+    .schema-json-name{
+        color:#000000D9;
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/schemaBox/style.scss:10]: ??

File: ui_src/src/domain/schema/components/schemaDetails/style.scss [hunk 4]
```diff
@@ -14,6 +14,10 @@ schema-details {
             .wrapper{
                 display: flex;
                 align-items: center;
+                .schema-json-name{
+                    color: #000000D9;
```
Comment (avrhamNeeman) [inline ui_src/src/domain/schema/components/schemaDetails/style.scss:18]: ??

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**